package.json

```diff
@@ -58,8 +53,8 @@
   },
   "exports": {
     "./chrome-launcher": "./chrome-launcher.js",
-    "./image-utils": "./image-utils.js",
-    "./page-objects": "./page-objects.js",
+    "./image-utils": "./image-utils/index.js",
+    "./page-objects": "./page-objects/index.js",
```

Comment:
Why is this change, but no actual changes in the files structure?

Reply:
Generate exports used to create the `image-utils.js` and `page-objects.js` files which would re-export from `./image-utils/index.js` and `./page-objects/index.js`

Reply:
Thanks for clarifying, all good.